tests/shared/src/test/scala/scala/meta/tests/quasiquotes/ErrorSuite.scala

```diff
@@ -1,13 +1,24 @@
+// TODO: Workaround for what seems to be a bug in ScalaTest Native 3.2.0-SNAP10.
+// If a test suite is defined in an empty package (i.e. outside any package declarations),
+// then it will spuriously fail to compile.
+// Discussion: https://github.com/scalameta/scalameta/issues/772#issuecomment-362380136
+
 // package scala.meta.tests
-// package quasiquotes
+package quasiquotes
 
 import org.scalatest._
 import org.scalameta.tests._
 import typecheckError.Options.WithPositions
 import compat.Platform.EOL
 
+// TODO: Workaround for what seems to be a bug in ScalaTest 3.2.0-SNAP10.
+// I had to remove $ characters from all test names in this file.
+// This is because ScalaTest seems to erroneously consider dollars to be name terminators,
+// so it would spuriously crash with "duplicated test" exceptions for e.g.:
+// test("...$ in Pat.Extract") { ... } and test("...$ in Pat.ExtractInfix")  { .. }.
```

Comment:
@bvenners Does this ring a bell? If it doesn't, I can try to minimize and submit a ticket.